.github/sync.yml

```diff
@@ -1,13 +1,10 @@
 group:
   - repos:
-      - SymbiFlow/fasm
-      - SymbiFlow/prjuray
-      - SymbiFlow/prjxray
-      - SymbiFlow/prjxray-bram-patch
-      - SymbiFlow/python-symbiflow-v2x
-      - SymbiFlow/symbiflow-arch-defs
-      - SymbiFlow/symbiflow-examples
-      - SymbiFlow/yosys-symbiflow-plugins
+      - chipsalliance/fpga-interchange-schema
```

Comment:
Can you please provide some context about the motivation to update some refs but remove others?

Reply:
In our sync meeting two weeks ago, we discussed only targeting repositories that have been fully moved over to chipsalliance and converted to the new license. These were the repositories proposed by @acomodi

Reply:
Thanks for explaining! You can add chipsalliance/f4pga-v2x as well.